website/pages/blog/graphqxl-language.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: GraphQXL language
```

Comment:
```suggestion
title: GraphQXL - The missing GraphQL Language Extension
```

Reply:
I would add a question mark to make it less "dictating"

Reply:
```suggestion
title: GraphQXL language - The missing GraphQL Language Extension?
```